Makefile

```diff
@@ -193,19 +193,31 @@ proto-lint:
 proto-check-breaking:
 	@$(DOCKER_BUF) breaking --against-input $(HTTPS_GIT)#branch=master
 
-build-docker:
-	docker build . -t archwayd:latest
-
-
-build-release: build-docker
-	mkdir -p $(CURRENT_DIR)/releases
-	docker run --rm --platform linux/amd64 -ti -v $(CURRENT_DIR)/releases:/root/.archway --entrypoint /bin/sh archwayd:latest -c "cp /usr/bin/archwayd /root/.archway"
-
 localnet:
 	docker-compose up
 
+release-dryrun:
+	$(DOCKER) run \
+		--rm \
+		-v "$(CURDIR)":/code \
+		-w /code \
+		-e LIBWASM_VERSION=$(LIBWASM_VERSION) \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		goreleaser/goreleaser-cross:v1.19.5 \
```

Comment:
introduce maybe a variable for goreleaser-cross version to easily update it in one place